src/themes/legacy.scss

```diff
@@ -1,3 +1,5 @@
 :root {
   --wxc-meeting-background: transparent;
+  --wxc-banner-background: transparent;
```

Comment:
let's hide this banner with a different technique:

```css
:root {
  .#{$WEBEX_COMPONENTS_CLASS_PREFIX}-banner.my-preview { display: none; }
}
```
and in the theme file, eg dark.scss
```js
:root .#{$WEBEX_COMPONENTS_CLASS_PREFIX}-theme-dark {
  .#{$WEBEX_COMPONENTS_CLASS_PREFIX}-banner.my-preview { display: block; }
}
```
we will remove these rules once the theming is complete.